Algorithmia/CLI.py

```diff
@@ -309,12 +308,10 @@ def list_languages(self, client):
         return table
 
     def getBuildLogs(self, user, algo, client):
-        api_response = client.algo(user + '/' + algo).build_logs()
-
-        if "error" in api_response:
```

Comment:
what will happen here if there is an error, now that you remove error handling?

Reply:
it will just raise the error as per the build_logs() endpoint, previously there was no error handling in the internal build_logs() endpoint, if it was a 500 we needed that to be handled here